TourMate/TourMate/TripViews/TripFromView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  TripFromView.swift
```

Comment:
I think the file name should be TripFormView?

Reply:
You're right, will make the change!